sdks/python/apache_beam/runners/interactive/extensions/interactive_beam_side_panel/src/index.ts

```diff
@@ -0,0 +1,41 @@
+// Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+import {
+  JupyterFrontEnd,
+  JupyterFrontEndPlugin
+} from '@jupyterlab/application';
+
+import { requestAPI } from './interactivebeamsidepanel';
+
+/**
+ * Initialization data for the interactive_beam_side_panel extension.
+ */
+const extension: JupyterFrontEndPlugin<void> = {
+  id: 'interactive-beam-side-panel',
+  autoStart: true,
+  activate: (app: JupyterFrontEnd) => {
+    console.log('JupyterLab extension interactive-beam-side-panel is activated!');
+
+    requestAPI<any>('get_example')
+      .then(data => {
+        console.log(data);
```

Comment:
Do you need this data log?

Reply:
I can remove this. The skeleton project is simply an initial state. I'm also thinking completely removing the server extension (python part) since we are not using it and that will introduce a third release story different from Beam SDK and the frontend extension.